backend/src/database/repositories/ResetTokenRepo.js

```diff
@@ -6,7 +6,7 @@ export default class ResetTokenRepo {
     };
 
     static getToken = async (token, userId) => {
-        return connection("tokens").where("userId", userId).first();
+        return connection("tokens").where("userId", userId).andWhere("token", token).first();
```

Comment:
Please undo this change. The andWhere clause doesn't fix the problem. It compounds the issue a bit

Reply:
Issues have been resolved. 

Source code is up to date.